guides/source/association_basics.md

```diff
@@ -1319,25 +1522,13 @@ class Book < ApplicationRecord
 end
 ```
 
+In this example, `Author` has many `books`, and each `Book` belongs to an `Author`. The `:inverse_of` option ensures that Rails correctly sets up the bidirectional association, allowing efficient and accurate access to related objects. For more details, see the [bi-directional association](#bi-directional-associations) section.
+
 ##### `:optional`
 
 If you set the `:optional` option to `true`, then the presence of the associated
 object won't be validated. By default, this option is set to `false`.
 
-##### `:polymorphic`
-
-Passing `true` to the `:polymorphic` option indicates that this is a polymorphic association. Polymorphic associations were discussed in detail <a href="#polymorphic-associations">earlier in this guide</a>.
-
-
-##### `:required`
-
-When set to `true`, the association will also have its presence validated. This will validate the association itself, not the id. You can use `:inverse_of` to avoid an extra query during validation.
-
-NOTE: required is set to `true` by default and is deprecated. If you don’t want to have association presence validated, use `optional: true`.
-
-##### `:strict_loading`
-
-Enforces strict loading every time the associated record is loaded through this association.
 
 ##### `:touch`
```

Comment:
The touch option also applies to most other associations.
Maybe we can make that a separate section as well?